src/features/projectsV2/ProjectDetails/components/ImageCarousel.tsx

```diff
@@ -1,11 +1,11 @@
-import React, { ReactElement, useEffect } from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
 import Stories from 'react-insta-stories';
 import getImageUrl from '../../../../utils/getImageURL';
 import { SliderImage } from '../../../projects/components/PlantLocation/ImageSlider';
 import { SingleCarouselImage } from './microComponents/SingleCarouselImage';
```

Comment:
Avoid using code from the old version of projects. Set up the SliderImage type within v2 itself.